### DISCUSSION TITLE
no discord server?

### DISCUSSION THREAD

**Discussion by tconley** [Budibase/budibase#3180] [General]
just found this amazing project. i see no discord server? is this thing already in a state where a company is running things and focusing on contributions mainly from internal hired devs or is there a need for a discord for the larger dev community?

**Comment by Fabian1410** [comment]
There is a discord Server, but for a better Issue Management they use GitHub.

**Reply by Fabian1410** [reply]
https://discord.gg/9HXuwceC

**Reply by mjashanks** [reply]
Yep - just like @Fabian1410 said. We don't want to do any Q&A on discord as things just get lost. GitHub Discussions is a much better medium for that.

The project is run by a company, with most of the work coming from internal, although we have a few fantastic external contributors.

We do use discord to talk to external contributors when needed. Apart from that, not much goes on in discord, as most stuff happens here.

**Comment by gaa23** [comment]
I was just looking for one when I found this. A good community really increases the value of the software that I love to use like AutoCode.com
